src/library_wasi.js

```diff
@@ -5,11 +5,22 @@
  */
 
 var WasiLibrary = {
-  proc_exit__deps: ['exit'],
   proc_exit__nothrow: true,
   proc_exit__sig: 'vi',
   proc_exit: function(code) {
```

Comment:
Ah, I was wondering how this even worked at first, but then I realized I was looking at an old baseline.

So we now use MUSL `_Exit` and `_exit` which both dispatch to this, correct?

Reply:
Yup